src/EventService.Models.Dto/Enums/AccessType.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
```

Comment:
And this 
![image](https://user-images.githubusercontent.com/43515136/215089616-07eb440e-b42f-4ba2-9b00-0096adc14a22.png)